sqlitecluster/SQLiteNode.cpp

```diff
@@ -241,6 +248,7 @@ void SQLiteNode::_replicate(SQLitePeer* peer, SData command, size_t sqlitePoolIn
                     ++commitAttemptCount;
                     result = _handleCommitTransaction(db, peer, command.calcU64("NewCount"), command["NewHash"]);
                     if (result != SQLITE_OK) {
+                        --_concurrentReplicateTransactions;
```

Comment:
I believe this will cause `_concurrentReplicateTransactions` to be decremented twice for a single transaction if `_handleCommitTransaction` returns anything other than `SQLITE_OK`.